src/test/java/org/apache/bcel/generic/CountVisitor.java

```diff
@@ -0,0 +1,3118 @@
+                && visitLNEGCount == that.visitLNEGCount
+                && visitLoadClassCount == that.visitLoadClassCount
+                && visitLoadInstructionCount == that.visitLoadInstructionCount
+                && visitLocalVariableInstructionCount == that.visitLocalVariableInstructionCount
+                && visitLOOKUPSWITCHCount == that.visitLOOKUPSWITCHCount
+                && visitLORCount == that.visitLORCount
+                && visitLREMCount == that.visitLREMCount
+                && visitLRETURNCount == that.visitLRETURNCount
+                && visitLSHLCount == that.visitLSHLCount
+                && visitLSHRCount == that.visitLSHRCount
+                && visitLSTORECount == that.visitLSTORECount
+                && visitLSUBCount == that.visitLSUBCount
+                && visitLUSHRCount == that.visitLUSHRCount
+                && visitLXORCount == that.visitLXORCount
+                && visitMONITORENTERCount == that.visitMONITORENTERCount
+                && visitMONITOREXITCount == that.visitMONITOREXITCount
+                && visitMULTIANEWARRAYCount == that.visitMULTIANEWARRAYCount
+                && visitNEWCount == that.visitNEWCount
+                && visitNEWARRAYCount == that.visitNEWARRAYCount
+                && visitNOPCount == that.visitNOPCount
+                && visitPOPCount == that.visitPOPCount
+                && visitPOP2Count == that.visitPOP2Count
+                && visitPopInstructionCount == that.visitPopInstructionCount
+                && visitPushInstructionCount == that.visitPushInstructionCount
+                && visitPUTFIELDCount == that.visitPUTFIELDCount
+                && visitPUTSTATICCount == that.visitPUTSTATICCount
+                && visitRETCount == that.visitRETCount
+                && visitRETURNCount == that.visitRETURNCount
+                && visitReturnInstructionCount == that.visitReturnInstructionCount
+                && visitSALOADCount == that.visitSALOADCount
+                && visitSASTORECount == that.visitSASTORECount
+                && visitSelectCount == that.visitSelectCount
+                && visitSIPUSHCount == that.visitSIPUSHCount
+                && visitStackConsumerCount == that.visitStackConsumerCount
+                && visitStackInstructionCount == that.visitStackInstructionCount
+                && visitStackProducerCount == that.visitStackProducerCount
+                && visitStoreInstructionCount == that.visitStoreInstructionCount
+                && visitSWAPCount == that.visitSWAPCount
+                && visitTABLESWITCHCount == that.visitTABLESWITCHCount
+                && visitTypedInstructionCount == that.visitTypedInstructionCount
+                && visitUnconditionalBranchCount == that.visitUnconditionalBranchCount
+                && visitVariableLengthInstructionCount == that.visitVariableLengthInstructionCount;
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(visitAALOADCount
```

Comment:
The comma should be at the end of each line, not the start. Note that you can use Eclipse to generate `hashCode()` and `equals()` methods for you which will guarantee that nothing is missing and that the formatting is not weird.

Reply:
I'm using IntelliJ to generate it. But it does not split line so I just split it.
Updated to put comma in end of line, please check